docs/api/wrapper/get.md

```diff
@@ -0,0 +1,19 @@
+## get
+
+Works just like [find](../find.md) but will throw an error if nothing matching
+the given selector is found. You should use `find` when searching for an element
+that may not exist. You should use this method when getting an element that should
+exist and it will provide a nice error message if that is not the case.
+
+```js
```

Comment:
Should we add an example showing the "happy path" for `get`? I mean, just getting an existing element in addition to showcase that it would throw otherwise.

Reply:
done